pyrasterframes/src/main/python/docs/raster-read.pymd

```diff
@@ -142,15 +144,17 @@ spark.read.raster(uri, lazy_tiles=False) \
     .select('proj_raster.tile').show(1, False)
 ```
 
-In the initial examples on this page, we used @ref:[`rf_tile`](reference.md#rf-tile) to explicitly request the realized tile from the lazy representation.
+In the initial examples on this page, you may have noticed that the realized (non-lazy) tiles are shown, but we did not change `lazy_tiles`. Instead, we used @ref:[`rf_tile`](reference.md#rf-tile) to explicitly request the realized tile from the lazy representation.
 
 ## Multiband Rasters
 
-A multiband raster represents a three dimensional numeric array. The first two dimensions are spatial, and the third dimension is typically designated as different bands. The bands could represent intensity of different wavelengths of light (or other electromagnetic radiation), or they could represent other phenomena such as measurement time, quality indications, or additional measurements.
+A multiband raster represents a three dimensional numeric array. The first two dimensions are spatial, and the third dimension is typically designated as different bands. The bands could represent intensity of different wavelengths of light (or other electromagnetic radiation), or they could represent other phenomena such as measurement time, quality indications, or additional measurements. 
+
+Multiband rasters files have a strictly ordered set of bands, which are typically indexed from 1. Some files have metadata tags associated with each band. Some files have a color interpetation metadata tag indicating how to interpret the bands.
```

Comment:
> which are typically indexed from 1

Do we need to be careful here in indicating that "band index" specifications in RasterFrames are zero based?

Reply:
I see that you point that out below, but I think further clarification that one-based is "colloquially" or "conventionally" in the context of EO dialog.

Reply:
My understanding of GDAL anyways is that they are defined as one-based index: 

https://gdal.org/doxygen/classGDALDataset.html#ad96adcf07f2979ad176e37a7f8638fb6 

My concern is the user that's a bit familiar with EO data does a `gdalinfo` and sees band 1 and 2 are what they need, then put `band_indexes=(1,2)`  


Maybe I'm missing your point?